tests/integration_test/data/jobs/hello-pt-cse/app/custom/train.py

```diff
@@ -27,9 +27,8 @@
 from nvflare.app_common.app_constant import ModelName
 
 # (optional) set a fixed location so we don't need to download everytime
-CIFAR10_ROOT = "/tmp/nvflare/data"
-
-MODEL_SAVE_PATH_ROOT = "/tmp/nvflare/workdir/cifar10"
+CIFAR10_ROOT = "~/data"
+MODEL_SAVE_PATH_ROOT = "~/data"
```

Comment:
so we want to save the model to the same location as the data? looks good otherwise

Reply:
yes, this is testing, we create and remove this folder after each test case.